server/app/views/admin/programs/ProgramApplicationView.java

```diff
@@ -172,11 +166,21 @@ private DivTag renderAnswer(long programId, AnswerData answerData) {
                     Styles.FLEX_AUTO, Styles.TEXT_RIGHT, Styles.FONT_LIGHT, Styles.TEXT_XS));
   }
 
-  private static SelectTag renderStatusOptionsSelector(StatusDefinitions statusDefinitions) {
+  private static DivTag renderStatusOptionsSelector(StatusDefinitions statusDefinitions) {
+    final String SELECTOR_ID = "status-selector";
```

Comment:
Up to you how much you want to care on this one. The ID is required because of HTML requirements. IDs have all kinds of opportunities for collisions since they need to be unique on a page.

We don't actually care about selecting on this ID ever nor do we really want to, I believe. What about postpending a UUID or random String to support this? We have RandomStringUtils or UUID. That way, we'd be pretty much guaranteed for there to be no collisions while still satisfying the requirement that there's an ID.

I'm aware that we don't follow this pattern elsewhere, but I think there's a lot of opportunity to move in that direction.

Reply:
done